scripts/fast_simulator_person

```diff
@@ -1,60 +0,0 @@
-#!/usr/bin/python
```

Comment:
This one was very much deprecated, right. 
Still, dont forget to mention them in your description